src/database/firestore.ts

```diff
@@ -0,0 +1,37 @@
+import { Firestore, DocumentReference } from "@google-cloud/firestore";
+
+export class FirestoreDB {
+    firestore: Firestore;
+    player_collection: DocumentReference;
+
+    constructor(projectId: string) {
+        this.firestore = new Firestore({
+            projectId: projectId,
+            keyFilename: process.env.GOOGLE_APPLICATION_CREDENTIALS,
+        });
+        this.player_collection = this.firestore.collection("players").doc("rocket_league");

```

Comment:
we can extract this logic into one higher level and this class will only handle db access.